src/modules/aws_acm/rpcs/request.ts

```diff
@@ -53,6 +53,13 @@ export class CertificateRequestRpc extends RpcBase {
     message: 'varchar',
   } as const;
 
+  inputTable: RpcInput = {
```

Comment:
I'm not completely sure, but not adding the `@internal` decorator is the same as having it? or it will be taken into account for the docs generation?

Reply:
This is how it's shown currently:
<img width="612" alt="image" src="https://user-images.githubusercontent.com/5357435/217857697-c8f56289-85df-4eed-8097-8c0a1e6f1fa7.png">

So you're right. I should add that decorator.

Reply:
oh nice, this is good to know!